src/services/persisters/s3Persister.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+const AWS = require('aws-sdk');
+
+const bucketName = "mirai-app";
+const MongoDbPersister = require('./persisters/mongoDbPersister');
```

Comment:
1. Don't reference services directly - use RootService, and
2. There's no use for the MongoDbPersister in the S3Persister.